extension/src/test/BaseAppTranslationFiles.test.ts

```diff
@@ -1,14 +1,18 @@
 import * as assert from "assert";
+import { writeFileSync } from "fs";
+import * as path from "path";
 import * as BaseAppTranslationFiles from "../externalresources/BaseAppTranslationFiles";
+import * as LanguageFunctions from "../LanguageFunctions";
 
 suite("Base App Translation Files Tests", function () {
-  const TIMEOUT = 360000;
+  const TIMEOUT = 360000; // Take some time to download blobs on Ubuntu... and windows!
```

Comment:
Haha, I also reacted on the fact that comment was spread all over the place - but I just felt that it might not worth mention 😁